book/chapters/chapter11/large-scale_benchmarking.qmd

```diff
@@ -139,7 +144,7 @@ odata$license
 c(nrow = odata$nrow, ncol = odata$ncol)
 ```
 
-If we want to access the actual data, then calling `$data` will download the data, import it into R, and then store the `data.frame` in the `OMLData` object:
+If we want to access the actual data, then accessing `$data` will download the data, import it into R, and then store the `data.frame` in the `OMLData` object:
```

Comment:
Is `accessing` right? We're saying "to access data then access `$data`", reads a bit strangely. If you don't like calling then what about:

```suggestion
If we want to access the actual data, then the `$data` field will download the data, import it into R, and then store the `data.frame` in the `OMLData` object:
```